packages/cli-kit/src/public/node/hooks/postrun.ts

```diff
@@ -1,10 +1,13 @@
+import {hook as deprecationsHook} from './postrun/deprecations.js'
 import {reportAnalyticsEvent} from '../analytics.js'
 import {outputDebug} from '../../../public/node/output.js'
 import {Hook} from '@oclif/core'
 
 // This hook is called after each successful command run. More info: https://oclif.io/docs/hooks
```

Comment:
Should we also call `clearNextDeprecationDate` in `prerun` in case a failed command run set it?